metadata.txt

```diff
@@ -17,7 +17,16 @@ repository=https://github.com/chairemobilite/transition_qgis
 # Recommended items:
 
 hasProcessingProvider=no
-changelog=Version 0.1 (initial)
+changelog=
```

Comment:
We have a duplication of the changelog here. We should see if we could simply import one file.

Reply:
I looked at a few qgis plugins and they all have the changelog in the metadata so it is available in the qgis plugin registry. The main CHANGELOG is for github and honestly easier to edit. I don't know if we can put a link in the metadata's changelog. If so, we could keep the latest release and add a link to the complete changelog on github. But I haven't seen a plugin with a changelog link, but I haven't checked them all.